apps/client/src/sitemap.xml

```diff
@@ -0,0 +1,303 @@
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/workshop</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/search</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/gathering-location</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/gc-supply</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/treasure-finder</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/levequests</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/log-tracker</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/alarms</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/alarm-group</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/alarms-overlay</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/fishing-reporter-overlay</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/mappy</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/mappy-overlay</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/list-panel-overlay</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/step-by-step-list-overlay</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/rotation-overlay</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/item-search-overlay</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
```

Comment:
Overlays and mappy should be removed as they aren't really available online and are made for the desktop app build.